README.md

```diff
@@ -12,11 +12,9 @@ It pulls all the cluster events and merge it with the existing data.
 | `ES_INDEX`          | Elasticsearch index name | assisted-service-events |
 | `LOGS_DEST`          | Logs directory, can be relative |  |
 | `ES_SERVER`          | Elasticsearch server address |  |
-| `ES_USER`          | Elasticsearch user name | elastic |
-| `ES_PASS`          | Elasticsearch password  |  |
+| `ES_USER(optional)`          | Elasticsearch user name | elastic |
+| `ES_PASS(optional)`          | Elasticsearch password  |  |
 | `REMOTE_SERVICE_URL` | Assisted service url  | https://api.openshift.com |
 | `OFFLINE_TOKEN` | Assisted service offline token  | |
-
-
-
-
+| `INVENTORY_URL` | Assisted service inventory url  | |
```

Comment:
can you put an example in the table?

Reply:
not my code, just documented it, don't have an example yet. will do after running it